src/main/java/javax/measure/unit/SI.java

```diff
@@ -146,23 +146,20 @@ public static SI getInstance() {
      * One radian is the angle between two radii of a circle such that the
      * length of the arc between them is equal to the radius.
      */
-    public static final AlternateUnit<Angle> RADIAN = si(new AlternateUnit<Angle>(
-            "rad", Unit.ONE));
+    public static final BaseUnit<Angle> RADIAN = si(new BaseUnit<>("rad"));
```

Comment:
This seems odd to me. Does this mean we can't convert between rads and degrees?

Reply:
Degrees are ultimately defined in terms of radians (in Jscience, a degree is 1/360th a revolution, which is 2pi radians), so this doesn't change that. It just changes the dimension of radians (and all derived angle units) to not be compatible with dimensionless units. After this PR, the only one of those that still exists is %, but this should reduce chance for error.